### ISSUE TITLE
Permissions are too restrictive for v0.5.0?

### ISSUE THREAD

**Issue by PChambino** [heroku/rack-timeout#129]
Hello! Thanks for the work in this simple and cool gem!

We just updated to the new version (v0.5.0) but it seems the gem permissions are too restrictive:
```
$ ll /var/lib/gems/2.4.0/gems/rack-timeout-0.5.0/lib/rack/timeout
total 40K
-rw------- 1 root root   78 May 21 16:07 base.rb
-rw------- 1 root root  13K May 21 16:07 core.rb
-rw------- 1 root root  920 May 21 16:07 logger.rb
-rw------- 1 root root 1.7K May 21 16:07 logging-observer.rb
-rw------- 1 root root  371 May 21 16:07 rails.rb
-rw------- 1 root root  134 May 21 16:07 rollbar.rb
drwxr-xr-x 2 root root 4.0K May 21 16:07 support
```

Which end ups raising a `LoadError: cannot load such file -- rack/timeout/base`. Could you have a look?

Related: https://github.com/rubygems/rubygems/issues/191 ?

**Comment by wuputah** [comment]
Thanks, made a rake task and released 0.5.0.1 to make sure this is addressed.

(There's no code changes to the gem itself, hence the "subpatch" version.)

**Comment by PChambino** [comment]
@wuputah Thanks!
